diagnostic.md

```diff
@@ -100,11 +133,23 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH http://www.example.com/donuts/42 \
+--header "Content-Type: application/json" \
+--data '{
+  "donut": {
+    "name": "Krüller"
+  }
+}'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function(data) {
+  return $.ajax({
+    url: 'http://wwww.example.com/donuts/' + data.donut.id,
```

Comment:
You don't need to use `data.donut.id` here, you can simply just use `donut.id` since you aren't replying on getting the donut data from a request